src/cluster_legacy.c

```diff
@@ -5546,6 +5546,31 @@ void clusterCloseAllSlots(void) {
     memset(server.cluster->importing_slots_from, 0, sizeof(server.cluster->importing_slots_from));
 }
 
+static void clusterDetermineClusterSize(int *reachable_primaries) {
```

Comment:
```suggestion
static int clusterDetermineClusterSize(void) {
```
Then return the size at the end.